test-cases/load/background-reads-overload.yaml

```diff
@@ -0,0 +1,18 @@
+test_duration: 30
```

Comment:
change the name of the file to: admission_control_overload_test.yaml (you can leave it under load folder)

Reply:
changed